app/Utils/feverUtil.php

```diff
@@ -1,69 +1,62 @@
 <?php
 
+declare(strict_types=1);
+
 class FreshRSS_fever_Util {
-	const FEVER_PATH = DATA_PATH . '/fever';
+	private const FEVER_PATH = DATA_PATH . '/fever';
 
 	/**
 	 * Make sure the fever path exists and is writable.
 	 *
-	 * @return boolean true if the path is writable, else false.
+	 * @return bool true if the path is writable, else false.
 	 */
-	public static function checkFeverPath() {
+	public static function checkFeverPath(): bool {
 		if (!file_exists(self::FEVER_PATH)) {
 			@mkdir(self::FEVER_PATH, 0770, true);
 		}
 
-		$ok = touch(self::FEVER_PATH . '/index.html');	// is_writable() is not reliable for a folder on NFS
-		if (!$ok) {
+		$isOk = touch(self::FEVER_PATH . '/index.html');	// is_writable() is not reliable for a folder on NFS
+		if (false === $isOk) {
```

Comment:
Please keep the boolean syntax. We are not going to change to such a verbose syntax everywhere

Reply:
Be careful when you just do "!" it's not a strict comparison 

see next comment please

Reply:
The negation of a strict boolean is also a strict boolean, so the goal for now is to be more strict with types, not to change boolean syntax (so no `=== true` or similar).

Reply:
A good practice is to always test what you want, not what you don't want. 
Adding an exclamation mark slows down the reading and increases the cognitive load.
If you test against what you want, there is less brain gymnastics to do, and you can more easily understand the code